File: templates/Uwp/_composition/_shared/Page.Ink.AddTransformService/Services/Ink/InkTransformResult.cs
using System.Collections.Generic;
using Windows.UI.Input.Inking;
using Windows.UI.Xaml;
using Windows.UI.Xaml.Controls;

namespace Param_ItemNamespace.Services.Ink
{
    public class InkTransformResult
    {
        public List<InkStroke> Strokes { get; set; } = new List<InkStroke>();

        public List<UIElement> TextAndShapes { get; set; } = new List<UIElement>();

        public Canvas DrawingCanvas { get; set; }

        public InkTransformResult(Canvas drawingCanvas)
        {
            DrawingCanvas = drawingCanvas;
        }
    }
}

File: templates/Uwp/_composition/_shared/Page.Ink.AddTransformServiceVB/Services/Ink/InkTransformResult.vb
Imports System.Collections.Generic
Imports Windows.UI.Input.Inking
Imports Windows.UI.Xaml
Imports Windows.UI.Xaml.Controls

Namespace Services.Ink
    Public Class InkTransformResult
        Public Property Strokes As List(Of InkStroke) = New List(Of InkStroke)()
        Public Property TextAndShapes As List(Of UIElement) = New List(Of UIElement)()
        Public Property DrawingCanvas As Canvas

        Public Sub New(drawingCanvas As Canvas)
            DrawingCanvas = drawingCanvas
        End Sub
    End Class
End Namespace
